tests/data/README.md

```diff
@@ -0,0 +1,33 @@
+- `make_empty_safe.sf` converts a full SDV SAFE folder into a 2.1 Mb folder of 1 vv-polarization annotation files.
+- `make_empty_img.py` Converts a measurement/ .tiff file into an all-zeros file < 50kb using SPARSE_OK=TRUE.
```

Comment:
Not sure if it is intentional, but I do see that this PR is trying to upload a lot of files into the repository. @LiangJYu can correct me if I am wrong, but I thought we decided to upload all the test data on `zenodo.org` in order to avoid increasing the size of the s1-reader repo. If this is the case, I would recommend to submit the unit test in a separate PR as this would require some more work (i.e. data uploading, set the CI) and we desperately need this PR for our CSLC-S1 Beta delivery